app/lib/features/news/widgets/news_side_bar.dart

```diff
@@ -53,7 +40,7 @@ class NewsSideBar extends ConsumerWidget {
         LikeButton(
           likeCount: news.likesCount().toString(),
           style: style,
-          color: fgColor,
+          color: Colors.white,
```

Comment:
I understand why that was removed, but the idea of the fg-color is to allow the user to set a higher contrast to whatever they have as the background (e.g. a very light image) to keep these accessible. We probably want to use any `fg-color` (if given) here in the future regardless from it not yet being implemented...

Reply:
Previously, `fgColor` was there based on the entire post and now it is based on the per slide mode. And also this side menus are applicable to entire post not for individual slides.

![Screenshot 2024-02-07 at 5 44 13 PM](https://github.com/acterglobal/a3/assets/51526480/aa98a028-1338-4abe-8f2c-c4055699097b)

Reply:
sure. future thing.